mockatests/wrappers/wrappers.c

```diff
@@ -60,6 +65,23 @@ __wrap_malloc(size_t size)
 	return NULL;
 }
 
+
+void *
+__wrap_calloc(size_t nmemb, size_t size)
+{
+	bool succeed = true;
+	fprintf(stderr, "In calloc\n");
```

Comment:
Are you sure this doesn't call malloc/calloc itself, resulting in infinite recursion?

Reply:
Most other 'fprintf' are commented out, perhaps this one should be too?

Reply:
It probably should but it's cosmetic tracing